client/src/components/History/CurrentCollection/CollectionPanel.vue

```diff
@@ -40,7 +40,7 @@ const dsc = computed(() => props.selectedCollections[props.selectedCollections.l
 const collectionElements = computed(() => collectionElementsStore.getCollectionElements(dsc.value, offset.value));
 const loading = computed(() => collectionElementsStore.isLoadingCollectionElements(dsc.value));
 const jobState = computed(() => dsc.value?.job_state_summary);
-const rootCollection = computed(() => props.selectedCollections[0]);
+const rootCollection = computed(() => props.selectedCollections[0] as HDCASummary);
```

Comment:
Is that always the HDCA ? Can we do something silly like making sure there's a `history_id` attribute there and throw an error if it is not?